00-open-a-shapefile.Rmd

```diff
@@ -36,40 +37,36 @@ After completing this activity, you will:
 
  * Know the difference between point, line, and polygon vector elements.
  * Understand the differences between opening point, line and polygon shapefiles
-    in R.
- * Understand the components of a spatial object (in R).
+    in `R`.
+ * Understand the components of a spatial object in `R`.
 
 ###Things You'll Need To Complete This Lesson
 
-You will need the most current version of R or RStudio loaded on your computer 
-to complete this lesson.
-
+You will need the most current version of `R` and, preferably, RStudio loaded on
+your computer to complete this lesson.
 
-###R Libraries to Install:
+####R Libraries to Install:
 
 * **raster:** `install.packages("raster")`
-* **sp:** `install.packages("sp")`
 * **rgdal:** `install.packages("rgdal")`
+* **sp:** `install.packages("sp")`
```

Comment:
sp loads with raster. curious as to the order change. not a big deal but it makes sense as it was given it's directly associated w raster. maybe i'm missing something?